### ISSUE TITLE
Is GPTQ working for pytorch? 

### ISSUE THREAD

**Issue by mikeseven** [sony/model_optimization#784]
### Issue Type

Bug

### Source

source

### MCT Version

Main

### OS Platform and Distribution

Ubuntu 20.04

### Python version

3.11

### Describe the issue

```shell
Using the quick start, GPTQ always leads to much inferior accuracy, from resnet to efficientnet classification torchvision models. I would have expected similar or better accuracy.
```

### Expected behaviour

```shell
Similar or better accuracy than PTQ.
Currently always much lower.
```

### Code to reproduce the issue

```shell
Quick start tutorial with resnet18/34/50,efficientnet b0 or v2 s.
GPTQ option.
```

**Comment by ofirgo** [comment]
@mikeseven 
It is possible that you got insufficient results with GPTQ because the quick start arguments for GPTQ didn't have compatible default values.
We fixed this (see #785), so you might want to try and run GPTQ again.

As a general note - the GPTQ algorithm performs differently for different networks, and its parameters might need different calibration. If the results are still insufficient, try to change the learning rate (`--gptq_lr`) or increase the number of GPTQ iterations (`--gptq_num_calibration_iter`). Note that the latter would increase the runtime of the algorithm.

**Comment by mikeseven** [comment]
Dear @ofirgo,

Indeed, I realized your fix improved previous results on efficientnet_b0 by about 1.1%. However, the main contributor is the batch size in the evaluation of hessian. Using MSE for both activations and weights does provide a slight improvement, which suggests that the quantization configuration for PTQ and GPTQ may need to be different.
I'll have to play with the parameters to figure out how to truly leverage it.

Thanks for your help.

**Comment by ofirgo** [comment]
@mikeseven
Thank you for the informative feedback.

I'm closing the issue since it seems that you figured that out. Let us know if you need any more help.
